pacemaker.spec.in

```diff
@@ -404,6 +410,10 @@ rm -rf %{buildroot}
 
 %post
 /sbin/chkconfig --add pacemaker || :
+%if %{with cman} && %{cman_native}
+# see https://bugzilla.redhat.com/1116401
+/usr/sbin/ccs_update_schema >/dev/null 2>&1 || :
+%endif
```

Comment:
System administrators often install packages without the intent of running them. Sometimes they may install a package just to see the man pages; sometimes they may install certain packages via an automated process such as puppet, and enable or disable the functionality on various nodes as needed. They don't expect the act of installing a package to irreversibly alter a file provided by another package. Configure time or start time is the appropriate place to do this, not install time. The clutter comes not merely from size, but from making someone who is trying to understand the spec have to refer to a BZ and a man page to figure out what's happening.